index.html

```diff
@@ -0,0 +1,968 @@
+            </div>
+          </div>
+
+          <div class="majordiv">
+            <h1>Amwell</h1>
+            <div class="div1">
+              <p>$100M</p>
+              <p class="grey">CORPORATE ROUND</p>
+            </div>
+            <div class="div2">
+              <p>
+                <i class="fas fa-tag"></i>
+                <a href="#">Information Technology</a>
+              </p>
+              <p>
+                <i class="fas fa-briefcase"></i>
+                <a href="#">Google Cloud Platform</a>
+              </p>
+              <div class="img">
+                <img src="images/divimg1.png" alt="divimg" />
+              </div>
+              <hr class="red" />
+              <p class="lastp">
+                Amwell Files For IPO And Raises $100M from Google Cloud (GOOG,
+                GOOGL)
+              </p>
+              <p class="grey lastp">on Pulse 2.0</p>
+            </div>
+          </div>
+          <div class="majordiv">
+            <h1>Amwell</h1>
+            <div class="div1">
+              <p>$100M</p>
+              <p class="grey">CORPORATE ROUND</p>
+            </div>
+            <div class="div2">
+              <p>
+                <i class="fas fa-tag"></i>
+                <a href="#">Information Technology</a>
+              </p>
+              <p>
+                <i class="fas fa-briefcase"></i>
+                <a href="#">Google Cloud Platform</a>
+              </p>
+              <div class="img">
+                <img src="images/divimg1.png" alt="divimg" />
+              </div>
+              <hr class="red" />
+              <p class="lastp">
+                Amwell Files For IPO And Raises $100M from Google Cloud (GOOG,
+                GOOGL)
+              </p>
+              <p class="grey lastp">on Pulse 2.0</p>
+            </div>
+          </div>
+
+          <div class="majordiv">
+            <h1>Amwell</h1>
+            <div class="div1">
+              <p>$100M</p>
+              <p class="grey">CORPORATE ROUND</p>
+            </div>
+            <div class="div2">
+              <p>
+                <i class="fas fa-tag"></i>
+                <a href="#">Information Technology</a>
+              </p>
+              <p>
+                <i class="fas fa-briefcase"></i>
+                <a href="#">Google Cloud Platform</a>
+              </p>
+              <div class="img">
+                <img src="images/divimg1.png" alt="divimg" />
+              </div>
+              <hr class="red" />
+              <p class="lastp">
+                Amwell Files For IPO And Raises $100M from Google Cloud (GOOG,
+                GOOGL)
+              </p>
+              <p class="grey lastp">on Pulse 2.0</p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- section 4 -->
+      <section class="grid2 alignment section4">
+        <div>
+          <div class="title">
+            <h1>Latest news</h1>
```

Comment:
- [ ] I know you did not worried about the content, but it is confusing to see the same title from a previous section again